streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -371,6 +372,9 @@ public boolean isStartingRunningOrPartitionAssigned() {
     private volatile KafkaFutureImpl<Uuid> restoreConsumerInstanceIdFuture = new KafkaFutureImpl<>();
     private volatile KafkaFutureImpl<Uuid> producerInstanceIdFuture = new KafkaFutureImpl<>();
 
+    // Missing source topic timeout tracking
```

Comment:
If you describe a member, I'd use a javadoc comment. But this comment isn't adding anything on top of the variable name, so maybe we can drop it altogether?